src/azure.js

```diff
@@ -1,8 +1,56 @@
+var _           = require('lodash');
 var azure       = require('fast-azure-storage');
 var api         = require('./v1');
 
 api.declare({
   method:     'get',
+  route:      '/azure/accounts',
+  name:       'azureAccounts',
+  input:      undefined,
+  output:     'azure-account-list-response.json#',
+  stability:  'stable',
+  scopes:     [['auth:azure-table-access:list-accounts']],
+  title:      "List Accounts Managed by Auth",
+  description: [
+    "Retrieve a list of all Azure accounts managed by Taskcluster Auth.",
+  ].join('\n')
+}, function(req, res) {
+  return res.reply({accounts: _.keys(this.azureAccounts)});
+});
+
+api.declare({
+  method:     'get',
+  route:      '/azure/:account/tables',
+  name:       'azureAccountTables',
+  query: {
+    nextTableName: /./,
```

Comment:
Again list call it continuation token... and do a proper regex: `/^[A-Za-z][A-Za-z0-9]{2,62}$/`.
The regex is actually from: https://docs.microsoft.com/en-us/rest/api/storageservices/fileservices/understanding-the-table-service-data-model